i2c/ina228_i2c/ina228_i2c.c

```diff
@@ -0,0 +1,125 @@
+#include <stdio.h>
+#include "pico/stdlib.h"
+#include "hardware/i2c.h"
+#include <math.h>
+
+// LED GPIOs
```

Comment:
```suggestion
// LED GPIO
```
:stuck_out_tongue_winking_eye: